packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -112,6 +112,23 @@ export const UIModeView: React.FC<{}> = ({
     });
   }, []);
 
+  React.useEffect(() => {
+    // Flood history to try to prevent back navigation
+    for (let i = 0; i < 20; i++)
+      window.history.pushState(undefined, '', window.location.href);
+
+    const onPopState = () => {
```

Comment:
Isn't this single popstate handler good enough for the basic scenario? I don't think we care that much about plugging every hole, but we can help users that accidentally use a "go back" shortcut once.